test/runtests.jl

```diff
@@ -2,4 +2,14 @@ using Libm
 using Base.Test
 
 # write your own tests here
-@test 1 == 2
+
+x = 2*rand(10000,1)-1
```

Comment:
The problem is that `erf()` is implemented differently in different ranges, so with this `x` you are not testing all code paths.

I'm really not competent to say what existing code to use as a starting point. I think it's worth to try a few alternative approaches. Taking advantage of extended CPU instructions is definitely something to consider.